hack/prow-e2e.sh

```diff
@@ -49,13 +49,13 @@ test-e2e-external-eks)
   TEST="external"
   export CLUSTER_TYPE="eksctl"
   ;;
-test-e2e-external-a1-eks)
-  TEST="external-a1-eks"
-  export K8S_VERSION_EKSCTL="1.30"
-  export INSTANCE_TYPE="a1.large"
+test-e2e-external-a1)
+  TEST="external-a1"
+  export AWS_AVAILABILITY_ZONES="us-west-2b,us-west-2c"
+  export INSTANCE_TYPE="a1.xlarge"
```

Comment:
Do we also need to update build-image.sh?

https://github.com/kubernetes-sigs/aws-ebs-csi-driver/blob/18aa7b714c3d1521a657e076344f95e22677704c/hack/e2e/build-image.sh#L73